app/views/catarse_bootstrap/user_notifier/mailer/_contribution_data.html.slim

```diff
@@ -24,11 +24,9 @@ p
   br/
   / We need the || for some old projects do not have the account info
   - owner = contribution.project.user
-  - owner_name = owner.try(:name)
-  - owner_document = owner.try(:cpf)
```

Comment:
The owner document number will be removed from all notifications templates that use this partial, this is correct?